codecov.yml

```diff
@@ -3,10 +3,10 @@ coverage:
     # PRs only
     patch:
       default:
-        threshold: 0.1%
+        threshold: 0.25%
```

Comment:
Because this PR deletes so many lines of code, despite every file in this PR having 100% code coverage, I was failing the codecov threshold check of `0.1` because my PR lowered overall code coverage by `0.12`.

I bumped the threshold to `0.25%`, because that's still a very small number and it gives us a bit of wiggle room going forward.

Reply:
This works for me 👍